packages/react-devtools-shared/src/__tests__/componentStacks-test.js

```diff
@@ -31,6 +32,12 @@ describe('component stack', () => {
     act = utils.act;
 
     React = require('react');
+    if (
+      React.version.startsWith('19') &&
+      React.version.includes('experimental')
+    ) {
+      supportsOwnerStacks = true;
+    }
```

Comment:
Can you try splitting these test cases and use pragmas?

```
// @reactVersion >= 19.0.1
// @gate enableOwnerStacks
```

Reply:
It's kind of annoying to have duplicate versions of every combination since it is only the format of the stack that changes. Especially since one of the tests was just added for newer versions in the first place. I've been using `gate(flags => enableOwnerStacks)` elsewhere as the pattern.

I wasn't sure about how these run but they run against builds too which makes the `@gate` not work right? Like if we're running at stable old 19 build versions it wouldn't work, right?

Reply:
I wasn't sure about how these run but they run against builds too which makes the @gate not work right?

Yeah, I just double-checked and it looks like we don't really support this case. When `@gate` is transformed by babel, the `@reactVersion` is filtered out, and we don't really check this.

The PR is incoming...

Reply:
Ugh, nevermind, we need to implement OR operator for these pragmas, but this is overcomplicated. You probably can just use `@gate experimental` here, since:
1. We don't test against `19.x.x` stable yet
2. If you don't use `@reactVersion` pragma, these tests are only going to be run against React from source == `experimental`.

Feel free to just skip this, not that important